apps/crn-server/src/controllers/research-output.controller.ts

```diff
@@ -144,6 +146,20 @@ export default class ResearchOutputController {
     const currentResearchOutput =
       await this.researchOutputDataProvider.fetchById(id);
 
+    let version: ResearchOutputVersionPostRequest | undefined;
+    if (researchOutputUpdateData.createVersion) {
+      version = {
+        title: currentResearchOutput?.title || '',
```

Comment:
shouldn't we send the new data here?

Reply:
No, when we make a new version that will replace the existing RO and the existing RO is transformed into the version.

Reply:
I knew I'd find the answer somehow